package-lock.json

```diff
@@ -5126,25 +5126,25 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "resolved": false,
```

Comment:
Not sure if we should commit these changes. Was this from an npm audit fix or just from an npm install?